pinot-server/src/main/java/org/apache/pinot/server/starter/helix/BaseServerStarter.java

```diff
@@ -520,7 +520,7 @@ private void setupHelixSystemProperties() {
    *
    * @param endTimeMs Timeout for the check
    */
-  private void startupServiceStatusCheck(long endTimeMs) {
+  private boolean startupServiceStatusCheck(long endTimeMs, ServerMetrics serverMetrics) {
```

Comment:
Is `serverMetrics` required here ? I dont see it being used.

Reply:
good catch. this is leftover from a previous iteration. I will remove it once we chat about the other comments